x-pack/plugins/lens/public/utils/native_renderer/native_renderer.test.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { render } from 'react-dom';
+import { NativeRenderer } from './native_renderer';
+
+describe('agnostic_renderer', () => {
```

Comment:
I was expecting you to explicitly define the React mocks, like `jest.mock('react');`. Does this test correctly find the `useEffect` redefinition you did?

Reply:
Having a `__mocks__` folder does the same thing: https://jestjs.io/docs/en/manual-mocks